core/src/test/java/org/keycloak/jose/jwk/JWKTest.java

```diff
@@ -60,6 +60,10 @@ public void publicRs256() throws Exception {
         assertNotNull(((RSAPublicJWK) jwk).getPublicExponent());
         assertNotNull(((RSAPublicJWK) jwk).getX509CertificateChain());
         assertEquals(PemUtils.encodeCertificate(certificate), ((RSAPublicJWK) jwk).getX509CertificateChain()[0]);
+        assertNotNull(((RSAPublicJWK) jwk).getSha1x509Thumbprint());
+        assertEquals(PemUtils.generateThumbprint(((RSAPublicJWK) jwk).getX509CertificateChain(), "SHA-1"), ((RSAPublicJWK) jwk).getSha1x509Thumbprint());
+        assertNotNull(((RSAPublicJWK) jwk).getSha256x509Thumbprint());
+        assertEquals(PemUtils.generateThumbprint(((RSAPublicJWK) jwk).getX509CertificateChain(), "SHA-256"), ((RSAPublicJWK) jwk).getSha256x509Thumbprint());
```

Comment:
Is it possible to test at least the length of thumbprint strings? I believe the length is always the same for SHA1 and SHA256 thumbprints? And it will help to recognize the mistake that thumbprints is not used at all (as mentioned above).